.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
   - 7.2
   - 7.3
```

Comment:
Add `7.4snapshot` here and update line below to `- if [[ $TRAVIS_PHP_VERSION != "7.4snapshot" ]]; then phpenv config-rm xdebug.ini; fi`

Reply:
Like so?
```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.2
   - 7.3
+  - 7.4snapshot'

 dist: trusty

@@ -21,8 +22,12 @@ env:
 matrix:
   fast_finish: true

+  include:
+    - php: 7.2
+      env: MINIMUMS=1 DEFAULT=1 DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
+
 before_install:
-  - if [[ $TRAVIS_PHP_VERSION != 7.3 ]]; then phpenv config-rm xdebug.ini; fi
+  - if [[ $TRAVIS_PHP_VERSION != "7.4snapshot" ]]; then phpenv config-rm xdebug.ini; fi

   - if [ -n "$GH_TOKEN" ]; then composer config github-oauth.github.com ${GH_TOKEN}; fi;
```

Reply:
Yes

Reply:
Except should have opening quote here `- '7.4snapshot'`...

Overall, `.travis.yml` in https://github.com/cakephp/acl/pull/159/files should be minimally changed, only lower/upper PHP versions changed

Reply:
Could you please post a complete travis.yml?